apps/site/components/Common/Partners/PartnerIcon/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Skeleton from '@node-core/ui-components/Common/Skeleton';
+import Tooltip from '@node-core/ui-components/Common/Tooltip';
+import type { ComponentProps, FC } from 'react';
+import { cloneElement } from 'react';
+
+import type { Partners } from '#site/types';
+
+import style from './index.module.css';
+import Button from '../../Button';
```

Comment:
```suggestion
import Button from '#site/components/Common/Button';
import type { Partners } from '#site/types';

import style from './index.module.css';
```
you can use alias import for button component